docs/source/benchmarking/fine_tuning_transformer_glue.rst

```diff
@@ -1,19 +1,29 @@
-Code in benchmarking/nursery/fine_tuning_transformer_glue
-=========================================================
+Code in benchmarking/examples/fine_tuning_transformer_glue
+==========================================================
 
 Selecting pre-trained transformer model from Hugging Face zoo and
-fine-tuning it to GLUE task.
+fine-tuning it to a GLUE task. This is in fact a whole family of benchmarks:
```

Comment:
Nice, learning about some stuff here that I didn't know existed. 

Is this AWS Glue or something else? https://aws.amazon.com/glue/

Reply:
No, this is a family of NLP benchmarks. I should add a comment with a reference.